### PR TITLE
docs: add 5.4.3 changelog

### PR DESCRIPTION
**Pull request by chenshuai2144**
## 5.4.3

`2023-04-19`

- 🐞 修复 FloatButton 警告: findDOMNode is deprecated in StrictMode.。[#41833](https://github.com/ant-design/ant-design/pull/41833) [@fourcels](https://github.com/fourcels)
- 🐞 修复 Layout.Sider 切换主题时存在背景切换延迟的问题。[#41828](https://github.com/ant-design/ant-design/pull/41828)
- 🌐 修复德语本地化文案。[#41780](https://github.com/ant-design/ant-design/pull/41780) [@aaarichter](https://github.com/aaarichter)
- 🐞 优化 Form 字段绑定，现在会忽略在 Form.Item 内的注释不再作为子组件进行绑定。&nbsp;。[#41771](https://github.com/ant-design/ant-design/pull/41771)
- 🐞 修复 Input.Password 在 Edge 中会额外展示切换按钮的问题。[#41759](https://github.com/ant-design/ant-design/pull/41759)
- 💄 修复 Space.Compact 包裹底层为 Tooltip 组件的组件时的样式问题。[#41707](https://github.com/ant-design/ant-design/pull/41707) [@foryuki](https://github.com/foryuki)
- TypeScript
  - 🤖 完善 Modal 的类型定义。[#41742](https://github.com/ant-design/ant-design/pull/41742) [@MuxinFeng](https://github.com/MuxinFeng)

----

## 5.4.3

`2023-04-19`

- 🐞 fix FloatButton warning: findDOMNode is deprecated in StrictMode. [#41833](https://github.com/ant-design/ant-design/pull/41833) [@fourcels](https://github.com/fourcels)
- 🐞 Fix Layout.Sider transition issue when switch theme. [#41828](https://github.com/ant-design/ant-design/pull/41828)
- 🌐 fixed typo in German locale. [#41780](https://github.com/ant-design/ant-design/pull/41780) [@aaarichter](https://github.com/aaarichter)
- 🐞 Optimize Form field binding, now will ignore comments in Form.Item as subcomponents. [#41771](https://github.com/ant-design/ant-design/pull/41771)
- 🐞 Fix Input.Password show additional toggle button in Edge. [#41759](https://github.com/ant-design/ant-design/pull/41759)
- 💄 Fix styling issues with components that have a Tooltip component as their base within the Space.Compact wrapper. [#41707](https://github.com/ant-design/ant-design/pull/41707) [@foryuki](https://github.com/foryuki)
- TypeScript
  - 🤖 Optimize the type definition of Modal. [#41742](https://github.com/ant-design/ant-design/pull/41742) [@MuxinFeng](https://github.com/MuxinFeng)

### CODE REVIEW

**Comment by afc163** [comment]
https://github.com/ant-design/ant-design/pull/41761

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ timeline: true
 
 - 🐞 修复 FloatButton 警告: findDOMNode is deprecated in StrictMode.。[#41833](https://github.com/ant-design/ant-design/pull/41833) [@fourcels](https://github.com/fourcels)
 - 🐞 修复 Layout.Sider 切换主题时存在背景切换延迟的问题。[#41828](https://github.com/ant-design/ant-design/pull/41828)
+- 🐞 修复 Tour 的 type 为 primary 时，箭头的颜色仍为白色的问题。[#41761](https://github.com/ant-design/ant-design/pull/41761)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:24]:
```suggestion
- 🐞 修复 Tour 的 `type="primary"` 时箭头的颜色仍为白色的问题。[#41761](https://github.com/ant-design/ant-design/pull/41761)
```

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ timeline: true
...
 - 🌐 修复德语本地化文案。[#41780](https://github.com/ant-design/ant-design/pull/41780) [@aaarichter](https://github.com/aaarichter)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:25]:
```suggestion
- 🇩🇪 修复德语本地化文案。[#41780](https://github.com/ant-design/ant-design/pull/41780) [@aaarichter](https://github.com/aaarichter)
```

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -21,6 +21,7 @@ timeline: true
 
 - 🐞 fix FloatButton warning: findDOMNode is deprecated in StrictMode. [#41833](https://github.com/ant-design/ant-design/pull/41833) [@fourcels](https://github.com/fourcels)
```
Comment (afc163) [inline CHANGELOG.en-US.md:22]:
```suggestion
- 🐞 fix FloatButton throws warning `findDOMNode is deprecated in StrictMode`. [#41833](https://github.com/ant-design/ant-design/pull/41833) [@fourcels](https://github.com/fourcels)
```

File: CHANGELOG.en-US.md [hunk 4]
```diff
@@ -15,6 +15,20 @@ timeline: true
...
 ---
 
+## 5.4.3
+
+`2023-04-19`
+
+- 🐞 fix FloatButton warning: findDOMNode is deprecated in StrictMode. [#41833](https://github.com/ant-design/ant-design/pull/41833) [@fourcels](https://github.com/fourcels)
+- 🐞 Fix Layout.Sider transition issue when switch theme. [#41828](https://github.com/ant-design/ant-design/pull/41828)
+- 🐞 Fix the problem that when the type of Tour is primary, the color of the arrow is still white. [#41761](https://github.com/ant-design/ant-design/pull/41761)
+- 🌐 fixed typo in German locale. [#41780](https://github.com/ant-design/ant-design/pull/41780) [@aaarichter](https://github.com/aaarichter)
```
Comment (afc163) [inline CHANGELOG.en-US.md:25]:
```suggestion
- 🇩🇪 Fix typo in German locale. [#41780](https://github.com/ant-design/ant-design/pull/41780) [@aaarichter](https://github.com/aaarichter)
```

**Comment by zombieJ** [comment]
CI 网络问题，重跑一下

**Comment by MadCcc** [comment]
package.json 回滚下？这个 format 之后看不出来哪里动了。。

**Comment by MadCcc** [comment]
再带一个这个 https://github.com/ant-design/ant-design/pull/41872

**Review by MadCcc — APPROVED** [review]

**Comment by MadCcc** [comment]
要 merge 一下 master？

File: CHANGELOG.zh-CN.md [hunk 5]
```diff
@@ -15,6 +15,21 @@ timeline: true
...
 
+## 5.4.3
+
+`2023-04-19`
+
+- 🐞 修复 FloatButton 警告: findDOMNode is deprecated in StrictMode.。[#41833](https://github.com/ant-design/ant-design/pull/41833) [@fourcels](https://github.com/fourcels)
+- 🐞 箭头元素兼容旧版本不支持 `clip-path: path()` 的浏览器。 [#41872](https://github.com/ant-design/ant-design/pull/41872)
+- 🐞 修复 Layout.Sider 切换主题时存在背景切换延迟的问题。[#41828](https://github.com/ant-design/ant-design/pull/41828)
+- 🐞 修复 Tour 的 `type="primary"` 时箭头的颜色仍为白色的问题。[#41761](https://github.com/ant-design/ant-design/pull/41761)
+- 🐞 优化 Form 字段绑定，现在会忽略在 Form.Item 内的注释不再作为子组件进行绑定。&nbsp;。[#41771](https://github.com/ant-design/ant-design/pull/41771)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:26]: 多了个 `&nbsp;。`

Comment (li-jia-nan) [reply CHANGELOG.zh-CN.md:26]: 火速蹭个 PR

Comment (li-jia-nan) [reply CHANGELOG.zh-CN.md:26]: - #41892

**Pull request merged**